Frontend/src/Test/GoogleMap.test.js

```diff
@@ -30,11 +27,7 @@ beforeEach(() => {
   };
 
   wrapper = ({ children }) =>
-    React.createElement(
-      APIProviderContext.Provider,
-      { value: mockContextValue },
-      children
-    );
+    React.createElement(APIProviderContext.Provider, { value: mockContextValue }, children);
```

Comment:
Nie lepiej było tak jak wyglądało to poprzednio? Tutaj jest jeszcze mało elementów ale w przypadku większej funkcji czytelniej jest trzymać się tego jak było.